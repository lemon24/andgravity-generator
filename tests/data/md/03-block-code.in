    indented
```
fenced
```
```python
hl
```
```python linenos
ln
```
```python linenos lineno-start=2
lns
```
```python lineno-start=2
lns2
```
```python emphasize-lines=1,3-
one
two
three
four
```
```python unknown
unknown
```
