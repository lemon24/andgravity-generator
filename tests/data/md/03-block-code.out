<pre class="code code-container"><code>indented
</code></pre>
<pre class="code code-container"><code>fenced
</code></pre>
<div class="highlight code-container"><pre class="code" data-lang="Python"><span></span><code><span class="n">hl</span>
</code></pre></div>
<div class="highlight code-container"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre class="code"><code><span class="normal">1</span></code></pre></div></td><td class="code"><div><pre class="code" data-lang="Python"><span></span><code><span class="n">ln</span>
</code></pre></div></td></tr></table></div>
<div class="highlight code-container"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre class="code"><code><span class="normal">2</span></code></pre></div></td><td class="code"><div><pre class="code" data-lang="Python"><span></span><code><span class="n">lns</span>
</code></pre></div></td></tr></table></div>
<div class="highlight code-container"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre class="code"><code><span class="normal">2</span></code></pre></div></td><td class="code"><div><pre class="code" data-lang="Python"><span></span><code><span class="n">lns2</span>
</code></pre></div></td></tr></table></div>
<div class="highlight code-container"><pre class="code" data-lang="Python"><span></span><code><span class="hll"><span class="n">one</span>
</span><span class="n">two</span>
<span class="hll"><span class="n">three</span>
</span><span class="hll"><span class="n">four</span>
</span></code></pre></div>
<div class="highlight code-container"><pre class="code" data-lang="Python"><span></span><code><span class="n">unknown</span>
</code></pre></div>
